spec/src/modules/tracker.js

```diff
@@ -24,11 +24,21 @@ const skipNetworkTimeoutTests = process.env.SKIP_NETWORK_TIMEOUT_TESTS === 'true
 const bundledDescriptionSuffix = bundled ? ' - Bundled' : '';
 const timeoutRejectionMessage = 'AbortError: This operation was aborted';
 const testAnalyticsTag = { param1: 'test', param2: 'test2' };
+const utmParameters = 'utm_source=attentive&utm_medium=sms&utm_campaign=campaign_1';
+const url = `http://localhost.test/path/name?query=term&category=cat&${utmParameters}`;
+const originReferrer = `localhost.test/path/name?${utmParameters}`;
+
+function validateOriginReferrer(requestParams) {
+  expect(requestParams).to.have.property('origin_referrer').to.equal(originReferrer);
```

Comment:
this overlaps the checks we do in L33-L35. Let's either keep this and remove the rest, or preferably replace it with 
```suggestion
  expect(requestParams).to.have.property('origin_referrer').to.contain('localhost.test/path/name');
```
so that it's more granular